lib/trailblazer/activity/dsl/linear/normalizer/output_tuples.rb

```diff
@@ -18,7 +18,7 @@ def to_a(*)
             # Connector when using Track(:success).
             class Track < Struct.new(:color, :adds, :options)
               def to_a(*)
-                search_strategy = options[:wrap_around] ? :WrapAround : :Forward
+                search_strategy = options&.key?(:wrap_around) ? :WrapAround : :Forward
```

Comment:
Is the point here to use the safe navigator `&.` or the `key?` method? Under which circumstances can `options` be `nil`?

Reply:
> Is the point here to use the safe navigator `&.` or the `key?` method?

Both

> Under which circumstances can `options` be `nil`?

I don't know it happens on application boot :/